pkg/minikube/assets/addons.go

```diff
@@ -311,7 +313,7 @@ var Addons = map[string]*Addon{
 			vmpath.GuestAddonsDir,
 			"olm.yaml",
 			"0640"),
-	}, false, "olm", map[string]string{
+	}, false, "olm", "OLM", map[string]string{
```

Comment:
this could be change t third party till we confirm

and also instead of OLM we could use their github org

https://github.com/operator-framework/

Reply:
we should create a follow up issue to contact the owners of those repos to see if they consent their org to be added offically as maintainer/responsible fr this addon, so in case olm image has anything malscious in it, they be point of contact for it ?

or maybe we could use the company name redhat for olm addon?

Wdyt  @ecordell @njhale @benluddy